sdk/dapp-kit/src/components/AccountDropdownMenu.css.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { style } from '@vanilla-extract/css';
```

Comment:
You can go down the rabbit hole by creating Tailwind-like systems, but IMO it adds a lot of complication in vanilla-extract and it didn't feel totally worth it at this point. That might change a little bit once I get more into theming